tox.ini

```diff
@@ -6,3 +6,9 @@ multi_line_output=3
 force_grid_wrap=0
 combine_as_imports=True
 lines_after_imports=2
+
+[flake8]
+ignore = E203,W503
```

Comment:
Why can't we write a code according to PEP rather than ignore rules?

Reply:
This changes were made automatically by black tool. According to [this](https://github.com/psf/black/issues/315) conversation I decided to ignore it.

Reply:
Got it.